backend/project.clj

```diff
@@ -16,6 +16,8 @@
                  [duct "0.8.2"]
                  [duct/hikaricp-component "0.1.0"]
                  [environ "1.1.0"]
+                 [garden "1.3.2"]
+                 [geo-clj "0.6.2"]
```

Comment:
Ah yes - two more stragglers from the client work. This is because 967 started as 901.